test/math/QuaternionTest.ooc

```diff
@@ -86,7 +85,7 @@ QuaternionTest: class extends Fixture {
 			expect(conjugate z == -quaternion5 z)
 		})
 		this add("normalized", func {
-			normalized := quaternion0 normalized
+			normalized := quaternion0 normalized as Quaternion
```

Comment:
That's because you removed the cast from the property. Why the cast was needed at all... I have no idea.